### PR TITLE
Add sample for `LabelFragment` in the `DatePicker` demo page (#2304)

### LINKED ISSUE #2304: Add a sample for the new  `LabelFragment` parameter on the demo page of the `BitDatePicker` component
Improve the `BitDatePicker` demo page. Add a sample for the new `LabelFragment` parameter and add a description about it on the demo page of the `BitDatePicker` component.

### PR DESCRIPTION
**Pull request by Md23Mh**
This closes #2304

### CODE REVIEW

**Review by maryamhdr — APPROVED** [review]

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor.cs [hunk 1]
```diff
@@ -297,4 +304,14 @@ private void HandleInvalidSubmit()
 {
     SuccessMessage = string.Empty;
 }";
+
+    #region Example Code 9
```
Comment (msynk) [inline src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor.cs:308]: please remove this `#region`

**Review by msynk — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@Md23Mh ?

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor.cs (hunk 1 above)
Comment (Md23Mh) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor.cs:308]: Could you explain the reason please? We have it in some of the demo pages and I think it helps to have a cleaner code.

Comment (msynk) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor.cs:308]: first of all, I don't like Tool specific features in an open source project.
secondly, you wrapped a single variable inside a region section which is redundant. you can simply fold the variable itself and have the same effect.

Comment (msynk) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor.cs:308]: btw, tnx for mentioning other usages of this feature. I'm going to submit a new issue to remove them :)

Comment (Md23Mh) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor.cs:308]: Sure I will remove it thank you

**Review by msynk — APPROVED** [review]

**Pull request merged**